__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/index");
+const db = require("../db/connection");
+const app = require("../app");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  describe("api calls", () => {
+    it("responds with status code of 200", () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+    it("reponds with an object containing an array", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.topics)).toBe(true);
+        });
+    });
+    it("array should be length of 3", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.topics.length).toBe(3);
+        });
+    });
+    it("each topic should be an object with 2 keys (slug + description), both having strings", () => {
+      return request(app)
```

Comment:
You could add all the expect statement above into a single test here. Your test suite can have multiple expects.